### DISCUSSION TITLE
Move beyond the image when zoomed in

### DISCUSSION THREAD

**Discussion by SpeeQz1** [PintaProject/Pinta#1143] [Ideas]
When trying to move the view so that you can see beyond the borders of the image the program doesn't let you, so I am suggesting to add the ability to move the camera as that would make precise cropping much easier mostly near the borders.

![image](https://github.com/user-attachments/assets/a82de8c7-aa58-4c40-9976-0d37c742e0c5)
![image](https://github.com/user-attachments/assets/a26c3989-c653-442e-8531-6d9ad6a15da9)

**Comment by cameronwhite** [comment]
This is probably related to issue #1027 where we also need the ability to scroll beyond the edge of the image
